integrationTest/test/babel/src/Casino.js

```diff
@@ -0,0 +1,28 @@
+var Bank = require('./Bank').default;
```

Comment:
I explicitly did not use an `import` to test if stryker-babel and stryker-javascript-mutator can deal with require statments instead of imports (modules vs scripts)

Reply:
Great reason! 👍

Reply:
I added a comment so we don't forget it